content/c-sharp/concepts/strings/terms/contains/contains.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.Contains()'
+Description: 'Defines whether a certain string *contains* a substring, passed as parameter.Returns a boolean data type.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Characters'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Contains()`** method defines whether the `string` includes a particular character or substring. Returns true if the character is included, else the method returns false. There is an opportunity to add extra parameters to correspond to the comparison rules.
```

Comment:
```suggestion
The **`.Contains()`** method determines whether a `string` includes a particular character or substring. It returns `true` if the character is included, otherwise the method returns `false`. There are additional parameters that can modify the comparison rules.
```